### DISCUSSION TITLE
Version 1 Alpha 3

### DISCUSSION THREAD

**Discussion by stuyam** [bootstrap-email/bootstrap-email#113] [General]
Two big PRs merged.
One is general style and utility tweaks: #111 
The other is more configuration to better support integrations: #112

<hr /><em>This discussion was created from the release <a href='https://github.com/bootstrap-email/bootstrap-email/releases/tag/1.0.0.alpha3'>Version 1 Alpha 3</a>.</em>

**Comment by pwoltman** [comment]
Could the container width be something that could pull from a sass configuration page and set via variable?  We have some emails that we set at 700px and 750px for layout purposes - 600px as a firm limit is too low, but then a full container-fluid is a little much.

**Reply by stuyam** [reply]
Yeah that is a good idea. The current container uses custom outlook conditional html table that I would have to figured out how to get that to be dynamic but I think it is worth looking into to customize. I agree 600px is small, it is sorta the "standard" for html emails though.
